Formula/g/gfxutil.rb

```diff
@@ -0,0 +1,46 @@
+class Gfxutil < Formula
+  desc "Device Properties conversion tool"
+  homepage "https://github.com/acidanthera/gfxutil"
+  url "https://github.com/acidanthera/gfxutil",
+    using:    :git,
+    tag:      "1.84b",
+    revision: "5284a662181ff4b81dd19d1279aedb68725b827f"
+  version "1.84b"
+  license :public_domain
+
+  depends_on xcode: :build
+  depends_on :macos
+
+  resource "edk2" do
+    url "https://github.com/acidanthera/audk",
+      using:    :git,
+      revision: "9b8affc22d154317ffce43fb675412c0dd92e790"
```

Comment:
```suggestion
    url "https://github.com/acidanthera/audk.git",
         revision: "9b8affc22d154317ffce43fb675412c0dd92e790"
```
Do we also need a git checkout here?

Also, can you add a comment indicating how to tell which `revision` to use when updating the formula?

Reply:
This one is a little more interesting.

Right now this checkout is pinned to the latest commit on `master`, which is how I got the `9b8affc` revision. This is because I didn't realize that the upstream is versioned with stable branches and not tags.

The latest stable branch is `audk-stable-202311`. There does exist a way to get the tarball for the branch in [GitHub's API](https://docs.github.com/en/rest/repos/contents?apiVersion=2022-11-28#download-a-repository-archive-tar), but it doesn't seem that there's a way to pin to an exact commit with this.

Would y'all prefer that I use the tarball link and always download the latest content from the stable branch, or continue using the checkout in order to pin the resource to today's latest commit (currently `cf294d6`) on the `audk-stable-202311` stable branch?